semeio/jobs/scripts/fm_pyscal.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+"""Forward model for connecting ERT with the Pyscal command line client"""
+import logging
+import sys
+import os
+import six
+
+from semeio.jobs.design_kw.design_kw import extract_key_value
+
+from pyscal import pyscalcli
+
+_logger = logging.getLogger("FM_PYSCAL")
+
+MAGIC_NONE = "__NONE__"
```

Comment:
What is `MAGIC_NONE`? If this is some well-known thing from pyscal, could you add that as a comment?

Reply:
Documented.